python/tutorials/01-vector-add.py

```diff
@@ -60,7 +61,6 @@ def add_kernel(x_ptr,  # *Pointer* to first input vector.
 def add(x: torch.Tensor, y: torch.Tensor):
     # We need to preallocate the output.
     output = torch.empty_like(x)
-    assert x.is_cuda and y.is_cuda and output.is_cuda
```

Comment:
I can rewrite this place something like this: 
```python
method_name = f"is_{DEVICE}"
assert getattr(x, method_name) and getattr(y, method_name) and getattr(output, method_name)
```

However, it seemed to me that there was no strong need for this check.

Reply:
this is a pretty useful assert as this tutorial is meant to be for beginners.

Reply:
I returned this check

Reply:
Hi @ThomasRaoux!

Should i do anything else in this PR?

Reply:
same comment as in https://github.com/triton-lang/triton/pull/5286. (maybe you can even merge those PRs)

Reply:
yes, I will merge them